docs/script_daily.md

```diff
@@ -66,7 +66,7 @@ DIFF_CMD=$(cat << END \
 # Assume that there are new bugs introduced, unless proven otherwise.
 echo -n "1" > "${WORKSPACE}/was_output.txt"
 
-eval "${DIFF_CMD}" | while read -r line
+eval "${DIFF_CMD}" 2>&1 | while read -r line
```

Comment:
Please update the daily script to use the json ouptut of the diff instead of the text output so it will be more stable in the future.

Reply:
We print the output of the diff command to the console and to a `${WORKSPACE}/bugs.txt` file. If we are using json output, we can't print it to the console/file because it is hard to read for the users.

Reply:
okay. So then keep it as is.